c2corg_ui/static/js/map/map.js

```diff
@@ -242,71 +253,114 @@ app.MapController.prototype.getVectorLayer_ = function() {
 
 
 /**
- * @param {number} scale
+ * @param {boolean} hilight
  * @return {ol.style.StyleFunction}
  * @private
  */
-app.MapController.prototype.createStyleFunction_ = function(scale) {
-  /**
-   * @type {Object.<string, ol.style.Icon>}
-   */
-  var iconCache = {};
-  /**
-   * @type {Object.<string, ol.style.Style|Array.<ol.style.Style>>}
-   */
-  var cache = {};
+app.MapController.prototype.createStyleFunction_ = function(hilight) {
   return (
       /**
        * @param {ol.Feature|ol.render.Feature} feature
        * @param {number} resolution
        * @return {ol.style.Style|Array.<ol.style.Style>}
        */
       function(feature, resolution) {
-        var type = /** @type {string} */ (feature.get('type'));
-        if (!type) {
-          // skip this feature
-          return null;
-        }
-
-        var id = /** @type {number} */ (feature.get('documentId'));
-        var key = type + id;
-        var style = cache[key];
-        if (!style) {
-          var iconKey = type + scale;
-          var icon = iconCache[iconKey];
-          if (!icon) {
-            icon = new ol.style.Icon(/** @type {olx.style.IconOptions} */ ({
-              scale: scale,
-              src: '/static/img/icons/' + type + '.png'
-            }));
-            iconCache[iconKey] = icon;
-          }
-
-          var text;
-          if (scale > 1 && typeof id !== undefined) { // on hover in list view
-            var title = /** @type {string} */(feature.get('title'));
-
-            text = new ol.style.Text({
-              text: title,
-              textAlign: 'left',
-              offsetX: 20,
-              font: 'bold 14px Calibri,sans-serif',
-              textBaseline: 'middle'
-            });
-          }
-
-          style = new ol.style.Style({
-            image: icon,
-            text: text
-          });
-          cache[key] = style;
+        // TODO: rename to document_type?
```

Comment:
OK, thought you might be referring to the top of the removed block:
`var type = /** @type {string} */ (feature.get('type'));`.

`Module` sounds good, it is the name of the feature property.